### ISSUE TITLE
[Component] [backtop] Backtop设置visibility-height为0时，显示问题

### ISSUE THREAD

**Issue by slient7** [element-plus/element-plus#13066]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.3.4`
- Element Plus Version: `2.3.5`
- Browser / OS: ` Windows: Chrome/112.0.5615.121`
- Build Tool: `Vite`

### Related Component

- `el-backtop`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZGl2IHN0eWxlPVwiaGVpZ2h0OiAxNTAwcHg7XCI+XG4gICAgPGVsLWJhY2t0b3AgOnZpc2liaWxpdHktaGVpZ2h0PVwiMFwiIC8+XG5cbiAgICA8cD5cbiAgICAgIDxlbC1pY29uIGNvbG9yPVwidmFyKC0tZWwtY29sb3ItcHJpbWFyeSlcIj48RWxlbWVudFBsdXMgLz48L2VsLWljb24+XG4gICAgICBFbGVtZW50IFBsdXMge3sgRXBWZXJzaW9uIH19ICsgVnVlIHt7IHZ1ZVZlcnNpb24gfX1cbiAgICA8L3A+XG4gIDwvZGl2PlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwiX28iOnt9fQ==)

### Steps to reproduce

<el-backtop :visibility-height="0" />

### What is Expected?

设置visibility-height为0或者为负数的时候，希望一直显示backtop

### What is actually happening?

需要页面滚动一下，才会显示出来

### Additional comments

_(empty)_

**Comment by slient7** [comment]
https://github.com/element-plus/element-plus/blob/dev/packages/components/backtop/src/use-backtop.ts
我看在onMounted()中已经显示修复了
```
// Give visible an initial value, fix #13066
    handleScroll()
```
但是在新下载下来的2.3.6版本包里，并没有handleScroll()这一句

**Comment by btea** [comment]
下一个版本发布之后才会包含。
